pulsar-client-shaded/pom.xml

```diff
@@ -86,6 +86,14 @@
                   <include>org.apache.httpcomponents:httpclient</include>
                   <include>commons-logging:commons-logging</include>
                   <include>org.apache.httpcomponents:httpcore</include>
+                  <include>org.apache.avro:avro</include>
```

Comment:
@jerrypeng there's also pulsar-client-kafka :)

Reply:
yup forgot you said that one as well